src/pages/Organization/OrganizationUsers.tsx

```diff
@@ -154,85 +159,86 @@ export default function OrganizationUsers({ id, navOrganizationId }: Props) {
           </div>
         ) : (
           <div className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-4">
-            {users?.results?.length === 0 ? (
-              <Card className="col-span-full">
-                <CardContent className="p-6 text-center text-gray-500">
-                  {t("no_users_found")}
-                </CardContent>
-              </Card>
-            ) : (
-              users?.results?.map((userRole) => (
-                <Card key={userRole.id} className="h-full">
-                  <CardContent className="p-4 sm:p-6 flex flex-col h-full justify-between">
-                    <div className="flex items-start gap-3">
-                      <Avatar
-                        name={`${userRole.user.first_name} ${userRole.user.last_name}`}
-                        imageUrl={userRole.user.profile_picture_url}
-                        className="h-12 w-12 sm:h-14 sm:w-14 text-xl sm:text-2xl flex-shrink-0"
-                      />
-
-                      <div className="flex flex-col min-w-0 flex-1">
-                        <div className="flex flex-col gap-1">
-                          <div className="flex items-start justify-between">
-                            <h1 className="text-base font-bold break-words pr-2">
+            {users?.results?.map((userRole) => (
```

Comment:
Why remove the CardContent that's displayed when there are no users? Only make additional changes if asked (or unless something is broken).